learning-mode.css

```diff
@@ -0,0 +1,282 @@
+  font-weight: 600;
+  text-decoration: underline !important;
+  background: none !important;
+  color: inherit !important;
+}
+.sensei-course-theme-lesson-actions .is-link:hover {
+  color: var(--sensei-primary-color) !important;
+}
+.sensei-course-theme-lesson-actions[aria-disabled=true], .sensei-course-theme-lesson-actions:disabled {
+  opacity: 0.5;
+  pointer-events: none;
+  cursor: not-allowed;
+}
+.sensei-course-theme-lesson-actions.is-completed.is-primary {
+  opacity: 0.8;
+  color: var(--bg-color) !important;
+  background: var(--text-color) !important;
+  border-color: var(--text-color) !important;
+}
+.sensei-course-theme-lesson-actions.is-completed.is-secondary {
+  opacity: 0.8;
+  color: var(--text-color) !important;
+  border-color: var(--text-color) !important;
+}
+.sensei-course-theme-lesson-actions.is-busy {
+  background-image: linear-gradient(-45deg, rgba(255, 255, 255, 0.5) 28%, rgba(255, 255, 255, 0.1) 28%, rgba(255, 255, 255, 0.1) 72%, rgba(255, 255, 255, 0.5) 72%);
+  animation: components-button__busy-animation 25000ms infinite linear;
+}
+.sensei-course-theme-lesson-actions.is-busy:disabled {
+  cursor: not-allowed;
+}
+.sensei-course-theme-lesson-actions .sensei-course-theme__button,
+.sensei-course-theme-lesson-actions .sensei-course-theme__link,
+.sensei-course-theme .sensei-course-theme__button,
+.sensei-course-theme .sensei-course-theme__link,
+.sensei-course-theme .wp-block-sensei-button .wp-block-button__link {
+  text-decoration: none;
+  border: solid 1px var(--sensei-primary-color);
+  display: block;
+  cursor: pointer;
+  text-transform: uppercase;
+  font-size: var(--wp--preset--font-size--small);
+  font-family: var(--wp--custom--button--typography--font-family);
+  border-radius: var(--wp--custom--button--radius);
+  padding-top: var(--wp--custom--button--spacing--padding--top);
+  padding-right: var(--wp--custom--button--spacing--padding--right);
+  padding-bottom: var(--wp--custom--button--spacing--padding--bottom);
+  padding-left: var(--wp--custom--button--spacing--padding--left);
+  font-weight: var(--wp--custom--button--typography--font-weight);
+  letter-spacing: var(--wp--custom--button--typography--letter-spacing);
+}
+.sensei-course-theme-lesson-actions .sensei-course-theme__button:focus,
+.sensei-course-theme-lesson-actions .sensei-course-theme__link:focus {
+  outline: dashed 1px var(--sensei-primary-color);
+  margin: -1px;
+  padding: 9px 12px;
+}
+.sensei-course-theme-lesson-actions .sensei-course-theme__button.has-icon,
+.sensei-course-theme-lesson-actions .sensei-course-theme__link.has-icon {
+  display: flex;
+  gap: 4px;
+  align-items: center;
+}
+.sensei-course-theme-lesson-actions .sensei-course-theme__button.has-icon svg,
+.sensei-course-theme-lesson-actions .sensei-course-theme__link.has-icon svg {
+  width: 20px;
+  height: 20px;
+}
+
+.sensei-course-theme__main-content > div.wp-block-group:last-child,
+.sensei-course-theme-lesson-actions {
+  flex-direction: row-reverse;
```

Comment:
I'm not sure CSS should change the order of the buttons. That brings up some accessibility issues (tab order won't change). The lesson actions block also makes sure the primary button is first. It also breaks alignment and pagination.

<img width="979" alt="image" src="https://user-images.githubusercontent.com/176949/203408748-a681fde4-3586-4117-9545-3dc403925919.png">

Original:

<img width="983" alt="image" src="https://user-images.githubusercontent.com/176949/203409166-6345dde7-c51f-4947-bdeb-ea981a54fd74.png">

Was this change intentional in the design @Luchadores?

Design:

<img width="610" alt="image" src="https://user-images.githubusercontent.com/176949/203409653-6620711e-a61e-48f8-9cb8-31a858169bfc.png">

Reply:
Removed it [here](https://github.com/Automattic/course/pull/68/commits/03a6ad31f10d97ba2b223ef70e7307262f5ecfbe) after discussion